OpenRA.Game/Widgets/Widget.cs

```diff
@@ -503,8 +503,7 @@ public virtual void HideChild(Widget child)
 
 		public virtual void RemoveChildren()
 		{
-			while (Children.Count > 0)
-				RemoveChild(Children[Children.Count - 1]);
+			Children.Clear();
```

Comment:
This means `Removed` is no longer called on the child widget.

Reply:
Good spotting. Fixed.